lib/pure/collections/sequtils.nim

```diff
@@ -113,11 +131,12 @@ proc distribute*[T](s: seq[T], num: int, spread = true): seq[seq[T]] =
   ##   assert numbers.distribute(6)[0] == @[1, 2]
   ##   assert numbers.distribute(6)[5] == @[7]
   assert(not s.isNil, "`s` can't be nil")
-  assert(num > 0, "`num` has to be greater than zero")
   if num < 2:
     result = @[s]
     return
 
+  let num = cast[int](num)
```

Comment:
OK. Done. I forgot to try if that works with a typecast and thought that the `cast` is a NOP anyway in the resulting code.